publications-2018.bib

```diff
@@ -663,6 +663,30 @@ @PhdThesis{MengyingXiaoPhD2018
   School                   = {Clemson University},
   Year                     = {2018},
   Url                      = {https://tigerprints.clemson.edu/cgi/viewcontent.cgi?article=3177}
+
+
+@article{doi:10.1093/gji/ggy466,
+  author = {Perry-Houts, Jonathan and Karlstrom, Leif},
+  title = {Anisotropic viscosity and time-evolving lithospheric instabilities due to aligned igneous intrusions},
+  journal = {Geophysical Journal International},
+  volume = {},
+  number = {},
+  pages = {ggy466},
+  year = {2018},
+  doi = {10.1093/gji/ggy466}
+}
+
+
+@Article{scott18:mta,
+  author =       {Scott Ladenheim and Yi-Chung Chen and Milan D. Mihajlovi{\'c} and Vasilis F. Pavlidis},
+  title =        {The {MTA}: {A}n Advanced and Versatile Thermal Simulator for Integrated Systems},
+  journal =      {IEEE Transactions on Computer-Aided Design of Integrated Circuits and Systems},
+  year =         2018,
+  volume =    37,
+  number =    12,
```

Comment:
Each of the `year`, `volume` and `number` entries likely need a to be encapsulated by braces.